src/UI/CustomHooks.re

```diff
@@ -0,0 +1,41 @@
+open Oni_Core;
+
+open Revery;
+open Revery.UI;
+
+module ColorEx = Utility.ColorEx;
+
+let colorTransition =
```

Comment:
Neat way to build on the `animation` hook 👍

Reply:
Should we upstream this to Revery too? It seems a little bit special-purpose, and should perhaps ideally be in some extended library for Revery.